### PR TITLE
[ZH] Update `Main_Page`

### PR DESCRIPTION
**Pull request by kj415j45**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Main_Page/zh.md [hunk 1]
```diff
@@ -29,25 +33,31 @@ outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
...
 # [谱面编辑](/wiki/Beatmap_Editor)
 
+osu! 的谱面都是社区玩家们的智慧结晶，想与大家分享你喜欢的歌却不知道从何下手？来学习一下如何利用游戏内的谱面编辑器吧。
+
 栏目：[Compose](/wiki/Beatmap_Editor/Compose) • [Design](/wiki/Beatmap_Editor/Design) • [Timing](/wiki/Beatmap_Editor/Timing) • [Song setup](/wiki/Beatmap_Editor/Song_Setup)
 
 组件：[AiMod](/wiki/Beatmap_Editor/AiMod) • [节拍细分](/wiki/Beatmap_Editor/Beat_Snap_Divisor) • [间距锁定](/wiki/Beatmap_Editor/Distance_Snap) • [菜单](/wiki/Beatmap_Editor/Menu) • [故事板负载](/wiki/Beatmap_Editor/SB_Load) • [时间轴](/wiki/Beatmap_Editor/Timelines)
 
-[作图](/wiki/Beatmapping) • [难度分级](/wiki/Beatmap/Difficulty) • [作图风格](/wiki/Mapping_Techniques) • [故事板](/wiki/Storyboard#storyboarding)
+活动：[作图](/wiki/Beatmapping) • [作图风格](/wiki/Mapping_Techniques) • [故事板](/wiki/Storyboard#storyboarding) • [皮肤](/wiki/Skinning)
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:53]: `Activities` -> `活动`
需要更好的译法

File: wiki/Main_Page/zh.md [hunk 2]
```diff
@@ -56,15 +66,47 @@ outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
 </div>
 <div class="wiki-main-page-panel">
 
-# 社区
+# 规章制度与法律术语
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:69]:
```suggestion
# 规章制度与法律条款
```

File: wiki/Main_Page/zh.md [hunk 3]
```diff
@@ -29,25 +33,31 @@ outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
...
 
-[谱面](/wiki/Beatmap) • [物件](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [分数](/wiki/Score) • [回放](/wiki/Replay) • [多人游戏](/wiki/Multi)
+[谱面](/wiki/Beatmap) • [难度](/wiki/Beatmap/Difficulty) • [物件](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [分数](/wiki/Score) • [回放](/wiki/Replay) • [奖章](/wiki/Medals) • [多人游戏](/wiki/Multi)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # [谱面编辑](/wiki/Beatmap_Editor)
 
+osu! 的谱面都是社区玩家们的智慧结晶，想与大家分享你喜欢的歌却不知道从何下手？来学习一下如何利用游戏内的谱面编辑器吧。
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:47]:
```suggestion
osu! 的谱面都是社区玩家们的智慧结晶，想与大家分享你喜欢的歌却不知道从何下手？来学习一下如何使用游戏内的谱面编辑器吧。
```

File: wiki/Main_Page/zh.md [hunk 4]
```diff
@@ -29,25 +33,31 @@ outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
 
 # 游戏玩法
 
+关于游戏物件、要素、机制与竞技内容。
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:36]:
```suggestion
osu! 的物件、要素、机制与竞技内容。
```

File: wiki/Main_Page/zh.md [hunk 5]
```diff
@@ -76,14 +118,20 @@ outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
...
-[API](/wiki/osu!api) • [机器人账户](/wiki/Bot_account) • [标识使用准则](/wiki/Brand_identity_guidelines)
+通过其他应用与 osu! 交互。
+
+[API](/wiki/osu!api) • [机器人账户](/wiki/Bot_account) • [标识使用准则](/wiki/Brand_identity_guidelines) • [osu!dev Discord 服务器](/wiki/osu!dev_Discord_server)
 
 </div>
 <div class="wiki-main-page-panel">
 
-# 关于 osu!wiki
+# Wiki and meta pages
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:128]:
```suggestion
# wiki 和其他页面
```

File: wiki/Main_Page/zh.md [hunk 6]
```diff
@@ -76,14 +118,20 @@ outdated_since: 1f00c369d836e214e9e2b5a3dc8ea21ff3392217
...
+
+osu! wiki 由志愿者编写并维护，提交建议、报告问题、撰写新文章均在 [osu-wiki 的 GitHub 存储库](https://github.com/ppy/osu-wiki)进行。
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:130]:
```suggestion
osu! wiki 由志愿者编写并维护。提交建议、报告问题、撰写新文章均在 [osu-wiki 的 GitHub 存储库](https://github.com/ppy/osu-wiki)进行。
```

**Comment by TicClick** [comment]
@kj415j45 what's the status of this PR? may it be merged as-is?

**Comment by kj415j45** [comment]
> @kj415j45 what's the status of this PR? may it be merged as-is?

If main page can be labeled as `no_native_review` then yes ¯\_(ツ)_/¯

**Comment by TicClick** [comment]
it can. what about your own suggestions, should they be applied?

**Comment by kj415j45** [comment]
> it can. what about your own suggestions, should they be applied?

I'll do a quick self-review right now and push the final commit on this PR.

File: wiki/Main_Page/zh.md [hunk 7]
```diff
@@ -1,25 +1,29 @@
...
+[规章制度](/wiki/Rules) • [安装](/wiki/Installation) • [注册](/wiki/Registration) • [帮助中心](/wiki/Help_Centre)
+
+[聊天面板](/wiki/Chat_Console) • [BanchoBot](/wiki/BanchoBot) • [常见问题](/wiki/FAQ) • [教程](/wiki/Guides) • [Glossary](/wiki/Glossary)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # 游戏客户端
 
+通过了解游戏的高度自定义系统和技术细节，深入 osu! 所展现的韵律与节奏世界。
```
Comment (kj415j45) [inline wiki/Main_Page/zh.md:25]:
```suggestion
通过了解游戏的多样化自定义系统和技术细节，深入 osu! 所展现的韵律与节奏世界。
```

**Comment by kj415j45** [comment]
should be ready to go.

**Review by TicClick — APPROVED** [review]

**Pull request merged**